plugins/woocommerce-blocks/tests/e2e/tests/products/products.block_theme.side_effects.spec.ts

```diff
@@ -67,7 +67,7 @@ test.describe( `${ blockData.name } Block `, () => {
 			postType: 'wp_template',
 		} );
 
-		await editor.canvas.click( 'body' );
+		await editor.canvas.locator( 'body' ).click();
```

Comment:
Similar to what I mentioned earlier, shouldn't we use `page.locator()` over `editor.canvas.locator()` consistently?

Reply:
Same here